opl/investigator/config.py

```diff
@@ -88,6 +92,12 @@ def load_config(conf, fp):
         conf.decisions_es_server = data["decisions"]["es_server"]
         assert not conf.decisions_es_server.endswith("/")
         conf.decisions_es_index = data["decisions"]["es_index"]
+        if "es_server_user" in data["decisions"]:
+            conf.decisions_es_server_user = data["decisions"]["es_server_user"]
+            conf.decisions_es_server_pass_env_var = data["decisions"][
+                "decisions_es_server_pass_env_var"
+            ]
+            conf.decisions_es_server_verify = data["decisions"]["es_server_verify"]
```

Comment:
Same here. Please move this outside of this loop and make it optional, defaulting to `True`.